pure.zsh

```diff
@@ -331,6 +331,16 @@ prompt_pure_async_git_arrows() {
 	command git rev-list --left-right --count HEAD...@'{u}'
 }
 
+prompt_pure_async_renice() {
```

Comment:
Can you add a code comment about what this does and why? For posterity.

Reply:
done